src/dotnet/APIView/ClientSPA/src/app/_components/revisions-list/revisions-list.component.ts

```diff
@@ -526,7 +526,7 @@ export class RevisionsListComponent implements OnInit, OnChanges {
         this.createRevisionInstruction = [
           `Run a <code>mvn package</code> build on your project, which will generate a number of build artifacts in the <code>/target</code> directory. In there, find the file ending <code>sources.jar</code>, and select it.`,
         ];
-        this.acceptedFilesForReviewUpload = ".sources.jar";
```

Comment:
We shouldn't change this. Java has .jar and .sources.jar. User must upload .sources.jar to create api review for Java.

Reply:
`sources.jar` is not actually an extension so it has to be `.jar` or else the upload will not work. The instruction still says upload `sources.jar`